avm/res/insights/webtest/main.bicep

```diff
@@ -0,0 +1,194 @@
+metadata name = 'Web Tests'
+metadata description = 'This module deploys a Web Test.'
+metadata owner = 'Azure/module-maintainers'
+
+@sys.description('Required. Name of the webtest.')
+param name string
+
+@sys.description('Required. User defined name if this WebTest.')
+param webTestName string
+
+@sys.description('Required. A single hidden-link tag pointing to an existing AI component is required.')
+param tags object
```

Comment:
Outstanding decision on how to action this with this resource type

Reply:
Your suggest is actually a pretty neat one. If the RT needs one specific tag of a certain type - then we can just have a mandatory parameter for that. @krbar what's your take on that?

Reply:
@ChrisSidebotham thanks for the suggestion, we can do it like this. The mandatory tag contains the `resourceId` of an Application Insights resource, so would create a mandatory parameter `appInsightResourceId`. 
However I suggest to add a short note to the description of the `tags` parameter informing the user about an additional tag.

CC: @AlexanderSehr 

This is the mandatory tag:
`'hidden-link:${appInsightResourceId}': 'Resource'`

Reply:
Like this? [541822f](https://github.com/Azure/bicep-registry-modules/pull/728/commits/541822f049fb695f1a28007325e85258900167d2)